scripts/install-mender.sh

```diff
@@ -209,10 +212,10 @@ add_repo() {
         echo "See https://docs.mender.io for updated APT repos information"
         exit 1
     fi
-    if test -f /etc/apt/sources.list.d/mender.list && \
-            grep -F "$repo_deprecated" /etc/apt/sources.list.d/mender.list >/dev/null; then
+    if test -f $MENDER_APT_SOURCES_LIST && \
```

Comment:
sorry I am being really annoying, we need double quotes around all refs. pre-approving. and thank you for applying all my comments!

Reply:
But why? We are defining this variable in the same script, we know exactly its value. Why do we need the quotes?

Reply:
hypothetical scenario: someone comes and for some reason changes this variable to "/my crazy dir/because" and we can be in trouble. this highly hypothetical person would have to check all the references, which she or he would probably not do.